docs/pages/project/contributing/contributing-docs.md

```diff
@@ -316,24 +318,19 @@ Just for fun, here are all the types:
 {% include alert.html type="primary" content="This is a primary alert" %}
 {% include alert.html type="secondary" content="This is a secondary alert" %}
 
-
-### Alerts
-
-{% include alert.html type="info" title="What is an alert?" content="An alert is a box that can stand out to indicate important information. You can choose from levels success, warning, danger, info, and primary. This example is an info box, and the code for another might look like this:" %}
+#### alert.html
 
 ```
```

Comment:
Don't include this description in box i.e remove stash.

Reply:
> ![Screenshot 2023-10-29 180116](https://user-images.githubusercontent.com/108334168/278870646-aa11802d-a367-45b0-8ac1-21d195c25aca.png) Don't include this description in box i.e remove stash.

okay got it